manifests/0.5.1/3scale-community-operator.v0.5.1.clusterserviceversion.yaml

```diff
@@ -125,16 +137,13 @@ spec:
 
     * Raising any issues you find using 3scale Operator
     * Fixing issues by opening [Pull Requests](https://github.com/3scale/3scale-operator/pulls)
-    * Improving [documentation](https://github.com/3scale/3scale-operator/blob/v0.5.0/doc/operator-user-guide.md)
+    * Improving [documentation](https://github.com/3scale/3scale-operator/blob/2.8-stable/doc/operator-user-guide.md)
```

Comment:
This should be 3scale-2.8-stable instead of 2.8-stable  (later is getting deprecated)